pkg/platform/runtime/artifactcache/artifactcache_test.go

```diff
@@ -0,0 +1,114 @@
+package artifactcache
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/ActiveState/cli/internal/fileutils"
+	"github.com/ActiveState/cli/internal/testhelpers/osutil"
+	"github.com/go-openapi/strfmt"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCache(t *testing.T) {
+	// Note: the length in bytes of each artifact is its index.
+	testArtifacts := []strfmt.UUID{
+		"000000000-0000-0000-0000-000000000000",
+		"74D554B3-6B0F-434B-AFE2-9F2F0B5F32BA",
+		"87ADD1B0-169D-4C01-8179-191BB9910799",
+		"5D8D933F-09FA-45A3-81FF-E6F33E91C9ED",
+		"992B8488-C61D-433C-ADF2-D76EBD8DAE59",
+		"2C36A315-59ED-471B-8629-2663ECC95476",
+		"57E8EAF4-F7EE-4BEF-B437-D9F0A967BA52",
+		"E299F10C-7B5D-4B25-B821-90E30193A916",
+		"F95C0ECE-9F69-4998-B83F-CE530BACD468",
+		"CAC9708D-FAA6-4295-B640-B8AA41A8AABC",
+		"009D20C9-0E38-44E8-A095-7B6FEF01D7DA",
+	}
+
+	dir, err := os.MkdirTemp("", "")
+	assert.NoError(t, err)
```

Comment:
Suggest changing assert to require, so that the test stops when the assertion fails.